sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -1,50 +1,50 @@
 <template>
-  <v-card class="mb-5">
-    <v-container fluid>
+  <body-header-container :enableCardView="true">
```

Comment:
A general point for the devs, I have a feeling that most of the time we use this with a card. Would be the case to set the property as `true` by default? Thoughts?
@ann-aot @dheepak-aot @andrepestana-aot @guru-aot

Reply:
True, I forgot that the full-page-container also has the v-card optionally.